src/utils/token.ts

```diff
@@ -39,23 +39,6 @@ export enum CurrencyTypeEnum {
     Token = 4,
 }
 
-export enum CurrencyIndexEnum {
-    KSM = 0,
-    USDT = 1,
-    XLM = 256,
-}
-
-export const currencyTokenSymbolMap: { [index: number]: string } = {
-    // XCM assets
-    0: 'KSM',
-    1: 'USDT',
-
-    // Stellar assets
-    256: 'XLM',
-}
-
-export const invertedTokenSymbolMap = invert(currencyTokenSymbolMap)
-
```

Comment:
So we don't need this anymore right? I just had a look and I found some leftover TODO comment from myself in the farming.ts file. Should we then remove the TODO comment etc as well?

Reply:
Yes I know the one you are referring to, over [here](https://github.com/pendulum-chain/pendulum-squids/blob/2c892a4cf063ef105dc85ca43b6f91e3ae60d354/src/utils/farming.ts#L211). Indeed we would not use the map so we can remove that too. 

Regarding the comment, I didn't understand it much because `invertedTokenSymbolMap` would map `string-> number` , given the definition of the [map](https://github.com/pendulum-chain/pendulum-squids/blob/2c892a4cf063ef105dc85ca43b6f91e3ae60d354/src/utils/token.ts#L48),  but this would have been accessed with `token0Symbol` which is a `u8` right? Maybe the comment was for an older map?

Reply:
I think I remember why we had this inverted map. Some time ago, our `CurrencyId` enum was not using `u8` indices to map to `XCM` currencies but instead we had a separate enum type for foreign currencies. The enum would then have the explicit token names like 'KSM', 'USDT', ... which is why this inverted map made sense.

Since this is not the case anymore, it's safe to remove.